templates/README.md

```diff
@@ -67,6 +67,12 @@ We literally have [*hundreds of terraform modules*][terraform_modules] that are
 {{ if has (ds "config") "usage" }}
 ## Usage
 
+**IMPORTANT:** The example shows pinning to the `master` branch. 
```

Comment:
This overall feels much wordier. Did you not like what we had before?

> **IMPORTANT:** The `master` branch is used in `source` just as an example. In your code, do not pin to `master` because there may be breaking changes between releases. Instead pin to the release tag (e.g. `?ref=tags/x.y.z`) of one of our [latest releases](https://github.com/cloudposse/terraform-aws-rds-cluster/releases).

Reply:
ok, reverted back.

I added more text because I felt it should describe three different things:
1. You should always do version pinning 
2. You should not pin to master
3. But we show pinning to master because we don't want to update all examples every time we update modules

But I agree less words is better.